casepro/statistics/models.py

```diff
@@ -55,6 +56,8 @@ def encode_scope(*args):
             return 'partner:%d' % args[0].pk
         elif types == [Org, User]:
             return 'org:%d:user:%d' % (args[0].pk, args[1].pk)
+        elif types == [Case]:
+            return 'case:%d' % (args[0].pk,)
```

Comment:
We might also want to add a `[Case, User]` tracking, or is that planned for another PR?